spec/index.html

```diff
@@ -6608,31 +6507,11 @@ <h6>Argument Compatibility Rules</h6>
                       <td>"b"</td>
                       <td>yes</td>
                     </tr>
-                    <tr>
```

Comment:
I removed these cases to avoid confusion, as readers might think they are different, while they are not.

> Or remove and have a respec "note" to remind the reader that "abc" is "abc"^^xsd:string.

I agree that this makes sense, will look into this.